examples/drupal_loadtest.rs

```diff
@@ -108,20 +116,20 @@ async fn drupal_loadtest_front_page(client: &mut GooseClient) {
 }
 
 /// View a node from 1 to 10,000, created by preptest.sh.
-async fn drupal_loadtest_node_page(client: &mut GooseClient) {
+async fn drupal_loadtest_node_page(client: &GooseClient) {
```

Comment:
@LionsAd `mut` is gone -- I still need to explore whether or not we can also get rid of `box::pin` as well when storing these futures.

Reply:
I was unable to find a way to get rid of the `box::pin` -- the problem is that `GooseTask`'s are stored in `GooseTaskSets` which are stored in `GooseAttacks` and the only method I've found for storing immutable objects without a `box::pin` has a ripple effect that I've been unable to get working. I think it should be left as a future optimization.